src/routes/(site)/events/event.svelte

```diff
@@ -29,6 +20,35 @@
   }
 
   onMount(() => {
+    const body: HTMLBodyElement | null = document.querySelector('body');
+    const copySvg: HTMLImageElement | null = document.querySelector('.copySvg');
+    if (copySvg) {
+      copySvg.src = body?.classList.contains('light')
+        ? '/assets/svg/copy-text-dark.svg'
+        : '/assets/svg/copy-text-light.svg';
+    }
+    const options = {
+      attributes: true,
+    };
+
+    function callback(mutationList: Array<MutationRecord>) {
+      mutationList.forEach((mutation: any) => {
```

Comment:
remove this any

Reply:
done in this [PR](https://github.com/EthanThatOneKid/acmcsuf.com/pull/629/commits/7497cb212b78689f075570e399b3c0c1eca3e257)